container/features/src/main/resources/features.xml

```diff
@@ -1363,6 +1363,7 @@
     <feature name="opennms-enlinkd-shell" description="OpenNMS :: Features :: Enlinkd :: Shell" version="${project.version}">
         <bundle>mvn:org.opennms.features.enlinkd/org.opennms.features.enlinkd.shell/${project.version}</bundle>
         <bundle>mvn:org.opennms.features.enlinkd/org.opennms.features.enlinkd.generator/${project.version}</bundle>
+        <bundle>mvn:org.opennms.features.topology.plugins.topo/org.opennms.features.topology.plugins.topo.linkd/${project.version}</bundle>
```

Comment:
not sure if we need this dependency - it doesn't solve the Karaf dependency issue (see https://issues.opennms.org/browse/HZN-1479?focusedCommentId=53785&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-53785)